src/_cffi_src/openssl/x509_vfy.py

```diff
@@ -338,4 +338,8 @@
 #else
 static const long Cryptography_HAS_X509_STORE_CTX_GET_ISSUER = 1;
 #endif
+
+#ifndef X509_V_FLAG_CB_ISSUER_CHECK
```

Comment:
What is this for?

Reply:
No idea. It's listed earlier in the file. Doesn't look like it's used.

Reply:
Looks like pyOpenSSL uses it.

If we need to handle it not existing, it needs to be in conditionals as well.

Reply:
1.1.0+ defines this as 0 in its deprecated paths so that's why it got added here. We should remove it from our bindings using `_conditional.py` (this also makes it easier for us to eventually know that we can entirely remove this binding when we only support 1.1.0+).

Reply:
@neheb Do you have time to make this fix so we can merge this?

Reply:
Not sure I did it right.

Reply:
You'll need to add a `Cryptography_HAS_X509_CB_ISSUER_CHECK` like you did for `Cryptography_HAS_DTLS1_METHOD`.

Reply:
Maybe now.